autoload/tsuquyomi/config.vim

```diff
@@ -105,12 +109,21 @@ function! tsuquyomi#config#tsscmd()
   return l:cmd
 endfunction
 
+function! s:system(cmd)
+  let out = ''
+  let job = job_start([&shell, &shellcmdflag, a:cmd], {'out_cb': {ch,msg->[execute("let out .= msg"), out]}, 'out_mode': 'raw'})
```

Comment:
I separated code for vim8 and vimproc. So this function will be called only for vim8